charts/palisade-service/templates/ingress-route.yaml

```diff
@@ -1,16 +1,17 @@
 apiVersion: traefik.containo.us/v1alpha1
 kind: IngressRoute
 metadata:
-    name: palisade-service
-    namespace: {{ include "palisade.namespace" . }}
+  name: palisade-service
+  namespace: {{ include "palisade.namespace" . }}
 spec:
-    entryPoints:
-        - web
-    routes:
-        - match: Host(`{{ .Values.global.hostname }}`) && PathPrefix(`{{ include "palisade-service.root" . }}/palisade`)
-          kind: Rule
-          services:
-              - name: palisade-service
-                port: 80
-          middlewares:
-              - name: palisade-pathparam-regex
\ No newline at end of file
+  entryPoints:
+    - web
+  routes:
+    - match: Host(`{{ .Values.global.hostname }}`) && PathPrefix(`{{ include "palisade-service.root" . }}/palisade`)
```

Comment:
Just a comment, but you've gone from whitespace of 4 spaces to whitespace of 2 spaces - is this a requirement of helm3 or is it a modification of your compiler? Or is this a symptom of a difference in enterprise intelliJ versus free intelliJ ?

Reply:
helm 3 is more strict